src/extension/features/accounts/calculate-irr/index.js

```diff
@@ -0,0 +1,250 @@
+import { Feature } from 'toolkit/extension/features/feature';
+import { isCurrentRouteAccountsPage } from 'toolkit/extension/utils/ynab';
+import { controllerLookup } from 'toolkit/extension/utils/ember';
+import { getEntityManager } from 'toolkit/extension/utils/ynab';
+
+export class CalculateIRR extends Feature {
+  injectCSS() {
+    return require('./index.css');
+  }
+
+  shouldInvoke() {
+    let { selectedAccount } = controllerLookup('accounts');
+    return (
+      isCurrentRouteAccountsPage() &&
+      selectedAccount &&
+      !selectedAccount.onBudget &&
+      ynabToolKit.options.CalculateIRR !== 'disabled'
+    );
+  }
+
+  invoke() {
```

Comment:
once you rebase on master, can you add a `destroy` method which  cleans up any elements your feature added? you should be able to go to the options page and `enable/disable` your feature without refreshing YNAB and see that it appears/disappears